### PR TITLE
upgrade: updated curated notebook to current standards

### PR DESCRIPTION
**Pull request by andrewferlitsch**
If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/automl/automl-text-classification.ipynb [hunk 1]
```diff
@@ -32,21 +32,22 @@
         "<table align=\"left\">\n",
```
Comment (ivanmkc) [inline notebooks/official/automl/automl-text-classification.ipynb:32]: <pre class="ql-syntax" spellcheck="false">Line #1.    delete_bucket = False
</pre><p>Actually, please revert to the previous form as we need to delete buckets in testing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/648/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

**Review by ivanmkc — CHANGES_REQUESTED** [review]
Left comment about cleanup.

File: notebooks/official/automl/automl-text-classification.ipynb (hunk 1 above)
Comment (andrewferlitsch) [reply notebooks/official/automl/automl-text-classification.ipynb:32]: @ivanmkc 

Not sure what you mean, we already have the delete bucket if testing:

if delete_bucket or os.getenv("IS_TESTING"):
    ! gsutil rm -r $BUCKET_URI

**Comment by andrewferlitsch** [comment]
Step #3: File ~/.local/lib/python3.9/site-packages/google/cloud/aiplatform/models.py:1147, in Endpoint._deploy_call(cls, api_client, endpoint_resource_name, model, endpoint_resource_traffic_split, deployed_model_display_name, traffic_percentage, traffic_split, machine_type, min_replica_count, max_replica_count, accelerator_type, accelerator_count, service_account, explanation_metadata, explanation_parameters, metadata, deploy_request_timeout, autoscaling_target_cpu_utilization, autoscaling_target_accelerator_duty_cycle)
Step #3:    1135 operation_future = api_client.deploy_model(
Step #3:    1136     endpoint=endpoint_resource_name,
Step #3:    1137     deployed_model=deployed_model,
Step #3:    (...)
Step #3:    1140     timeout=deploy_request_timeout,
Step #3:    1141 )
Step #3:    1143 _LOGGER.log_action_started_against_resource_with_lro(
Step #3:    1144     "Deploy", "model", cls, operation_future
Step #3:    1145 )
Step #3: -> 1147 operation_future.result()
Step #3: 
Step #3: File ~/.local/lib/python3.9/site-packages/google/api_core/future/polling.py:137, in PollingFuture.result(self, timeout, retry)
Step #3:     132 self._blocking_poll(timeout=timeout, **kwargs)
Step #3:     134 if self._exception is not None:
Step #3:     135     # pylint: disable=raising-bad-type
Step #3:     136     # Pylint doesn't recognize that this is valid in this case.
Step #3: --> 137     raise self._exception
Step #3:     139 return self._result
Step #3: 
Step #3: InternalServerError: 500 INTERNAL

**Comment by andrewferlitsch** [comment]
Notebook testing log says cancelled after 222 minutes. No explanation, no failures in log. Make command decision to merge PR due to notebook scaling issues.

**Pull request merged**